services/movie_session.py

```diff
@@ -42,3 +42,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = get_movie_session_by_id(movie_session_id).ticket_set.all()
+
+    return [{"row": ticket.row, "seat": ticket.seat} for ticket in tickets]
```

Comment:
Interesting decision, I like it

Reply:
it seems to me, u could use `values` method here